explorer/src/lib/components/statistics-panel/StatisticsPanel.css

```diff
@@ -0,0 +1,104 @@
+.statistics-panel {
+  display: flex;
+  padding: 1rem 1.375rem;
+  flex-direction: column;
+  column-gap: 0.625rem;
+  row-gap: 0.75rem;
+  border-radius: 1.5rem;
+  background-color: var(--surface-color);
+}
+
+.statistics-panel__statistics {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 1.5rem;
+  overflow: hidden;
+}
+
+.statistics-panel__statistics-item {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.3125rem;
+  position: relative;
+  justify-content: center;
+}
+
+.statistics-panel__statistics-item::after,
+.statistics-panel__statistics-item::before {
+  content: "";
+  position: absolute;
+  background-color: var(--primary-color);
+  z-index: 1;
+}
+
+.statistics-panel__statistics-item::after {
+  inline-size: 100vw;
+  block-size: 1px;
+  inset-inline-start: 0;
+  inset-block-start: calc(calc(1.5rem / 2) * -1);
+}
+
+.statistics-panel__statistics-item::before {
+  inline-size: 1px;
+  block-size: 100%;
+  inset-block-start: 0;
+  inset-inline-start: calc(calc(1.25rem / 2) * -1);
+}
+
+.statistics-panel__statistics-item > div {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  column-gap: 0.3125rem;
+}
+
+.statistics-panel__statistics-item-value {
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+.statistics-panel__statistics-item-title {
+  text-transform: uppercase;
+  font-size: 0.75rem;
+}
+
+.statistics-panel__world-map {
+  padding-top: 1.25rem;
+  border-top: 1px solid var(--primary-color);
+}
+
+.statistics-panel__world-map img {
+  display: block;
+  margin: 0 auto;
+}
+
+@media (min-width: 768px) {
+  .statistics-panel__statistics {
+    grid-template-columns: repeat(4, 1fr);
+    gap: 1.25rem;
+  }
+
+  .statistics-panel {
+    flex-direction: row;
+  }
+
+  .statistics-panel__statistics-item::after {
+    inset-block-start: calc(calc(1.25rem / 2) * -1);
+  }
+
+  .statistics-panel__statistics-item-value {
+    font-weight: 500;
+    font-size: 1.125rem;
+  }
+
+  .statistics-panel__statistics-item-title {
+    text-transform: uppercase;
```

Comment:
You don't need this.

Reply:
This is a safe net in case and it's better to uppercase with css then to do it in the markup.

Reply:
No, but this is already defined on line 61. So you don't need to overwrite it.

Reply:
You're right.